csharp/Momento.csproj

```diff
@@ -2,6 +2,16 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
+    <PackageId>Momento.Protos</PackageId>
+    <Authors>Momento</Authors>
+    <Company>Momento Inc</Company>
+    <Description>
+      Generated gRPC and Protocol Buffers types for Momento clients
+    </Description>
+    <PackageTags>caching, cache, serverless, key value, simple caching service</PackageTags>
+    <Copyright>Copyright (c) Momento Inc 2022</Copyright>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <ProjectUrl>https://github.com/momentohq/client_protos/tree/main/csharp</ProjectUrl>

```

Comment:
The URL should just be to /client_protos/

Reply:
I modeled that after the pypi info, but I tend to agree with you. I've changed it.